Makefile

```diff
@@ -80,10 +80,13 @@ help:  ## Display this help
 .PHONY: testprereqs
 testprereqs: $(KUBEBUILDER) $(KUSTOMIZE)
 
-.PHONY: test
-test: testprereqs generate fmt lint ## Run tests
+.PHONY: unit
+unit: testprereqs ## Run tests
 	source ./hack/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./controllers/... ./ipam/... -coverprofile ./cover.out; cd $(APIS_DIR); go test -v ./... -coverprofile ./cover.out
 
+.PHONY: test  ## Run formatter, linter and tests
+test: fmt lint unit
```

Comment:
I would still want to keep test as is i.e.  Keep generate

Reply:
Good point. Fixed!